apps/annotation/CMakeLists.txt

```diff
@@ -1,19 +1,21 @@
-SET(deps opencv_core opencv_highgui opencv_imgproc)
-ocv_check_dependencies(${deps})
+SET(OPENCV_ANNOTATION_DEPS opencv_core opencv_highgui opencv_imgproc)
+ocv_check_dependencies(${OPENCV_ANNOTATION_DEPS})
 
 if(NOT OCV_DEPENDENCIES_FOUND)
    return()
 endif()
 
 project(annotation)
+set(the_target opencv_annotation)
 
-ocv_include_directories("${CMAKE_CURRENT_SOURCE_DIR}" "${OpenCV_SOURCE_DIR}/include/opencv")
-ocv_include_modules(${deps})
+ocv_target_include_directories(${the_target} PRIVATE "${CMAKE_CURRENT_SOURCE_DIR}" "${OpenCV_SOURCE_DIR}/include/opencv")
```

Comment:
@alalek it seems that these ocv lines are also unrecognized by the buildbot.
Should I delete them as well?